src/cli/mod.rs

```diff
@@ -75,7 +78,28 @@ fn init_logger(opts: LogOptions) {
         .format_module_path(opts.module_path)
         .filter_level(log::LevelFilter::Info)
         .filter_module("witnet", opts.level)
-        .init();
+        .filter_module("witnet_node", opts.level);
+
+    // Initialize Sentry (automated bug reporting) if explicitly enabled in configuration
+    if opts.sentry_telemetry {
+        // Initialize client
+        let guard = sentry::init(
+            "https://def0c5d0fb354ef9ad6dddb576a21624@o394464.ingest.sentry.io/5244595",
```

Comment:
Would it make sense to be able to change this URL in witnet.toml?

Reply:
Let's postpone the decision until we have validated if Sentry is useful and how long it stays here.